### ISSUE TITLE
Error when training in COCO datasets with"th train.lua"

### ISSUE THREAD

**Issue by Soledad89** [facebookresearch/deepmask#43]
When I run the command without any changes following the tutorial :
# ~/Documents/deepmask$ th train.lua
then, it goes: 
-- ignore option rundir	
-- ignore option dm	
-- ignore option reload	
-- ignore option gpu	
-- ignore option datadir	
| running in directory /home/w00378682/Documents/deepmask/exps/deepmask/exp	
| number of paramaters trunk: 15198016	
| number of paramaters mask branch: 1608768	
| number of paramaters score branch: 526337	
| number of paramaters total: 17333121	
convert: data//annotations/instances_train2014.json --> .t7 [please be patient]
convert: data//annotations/instances_train2014.json --> .t7 [please be patient]
FATAL THREAD PANIC: (pcall) not enough memory
/home/wang/torch/install/bin/luajit: ...wang/torch/install/share/lua/5.1/threads/threads.lua:183: [thread 2 callback] not enough memory
stack traceback:
	[C]: in function 'error'
	...wang/torch/install/share/lua/5.1/threads/threads.lua:183: in function 'dojob'
	...wang/torch/install/share/lua/5.1/threads/threads.lua:264: in function 'synchronize'
	...wang/torch/install/share/lua/5.1/threads/threads.lua:142: in function 'specific'
	...wang/torch/install/share/lua/5.1/threads/threads.lua:125: in function 'Threads'
	/home/wang/Documents/deepmask/DataLoader.lua:40: in function '__init'
	/home/wang/torch/install/share/lua/5.1/torch/init.lua:91: in function </home/wang/torch/install/share/lua/5.1/torch/init.lua:87>
	[C]: in function 'DataLoader'
	/home/wang/Documents/deepmask/DataLoader.lua:21: in function 'create'
	train.lua:101: in main chunk
	[C]: in function 'dofile'
	...wang/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:145: in main chunk
	[C]: at 0x00405d50

Can somebody help me with this issue? I have configured deepmask with Ubuntu16.04 and GTX1080, cuda8.0 and 32GB memory. Thanks sooooo much.

**Comment by tongerpeng** [comment]
I met the same error~~~

**Comment by tongerpeng** [comment]
Did you solved this problem??

**Comment by Soledad89** [comment]
@tongerpeng  Nope......

**Comment by tylin** [comment]
duplicate https://github.com/facebookresearch/deepmask/issues/14
